### ISSUE TITLE
bug on float with scientific notation

### ISSUE THREAD

**Issue by yannickbattail** [hugheaves/scadformat#6]
float with scientific doesn't seams to work
```scad
c=8.014786e+00;
d=8.014786E+00;
e=8.014786E-00;
f=8.014786e-00;
g=8.014786E00;
h=8.014786e00;
i=9.9e-126;
j=9.e-126;
k=9e-126;
l=9e-126;
m=-9e-126;
```

It display the error:
```
line 1:10 missing ';' at 'e'
ERROR   syntax error on line 1:10 - missing ';' at 'e'
```

**Comment by yannickbattail** [comment]
I tried this, but i doesn't work
```antlr
FLOAT
:
    (
        (DIGIT+ '.' DIGIT+)
        | (DIGIT+ '.')
        | ('.' DIGIT+)
    )
    (
        ( 'e' | 'E' )
        ( PLUS | MINUS )?
        DIGIT+
    )?
;
````

**Comment by hugheaves** [comment]
Not sure why your new FLOAT rule didn't work - I tried it and it actually worked for me.
FWIW, I went and and fixed this in the v0.5 release. Give it a try and let me know if it works for you.
